src/network_manager.rs

```diff
@@ -72,35 +72,26 @@ impl ActiveConnections {
 }
 
 // Send some data to a peer that we didn't accept his connection
-pub type FallbackFunction = dyn Fn(
-        &KeyPair,
-        &mut Endpoint,
-        &HashMap<SocketAddr, TransportType>,
-        &MessageHandlers,
-    ) -> PeerNetResult<()>
+pub type FallbackFunction = dyn Fn(&KeyPair, &mut Endpoint, &HashMap<SocketAddr, TransportType>) -> PeerNetResult<()>
     + Sync;
-pub type HandshakeFunction = dyn Fn(
-        &KeyPair,
-        &mut Endpoint,
-        &HashMap<SocketAddr, TransportType>,
-        &MessageHandlers,
-    ) -> PeerNetResult<PeerId>
+pub type HandshakeFunction = dyn Fn(&KeyPair, &mut Endpoint, &HashMap<SocketAddr, TransportType>) -> PeerNetResult<PeerId>
```

Comment:
This function isn't useful anymore is it ? It should have been removed with the HandshakeHandler refactory I guess

Reply:
Removed